app/models/consultation.rb

```diff
@@ -151,7 +158,7 @@ def end_date_from(now = Time.zone.now)
     # Letters are printed at 5:30pm and dispatched the next working day (Monday to Friday)
     # Second class letters are delivered 2 days after they’re dispatched.
     # Royal Mail delivers from Monday to Saturday, excluding bank holidays.
-    default_start_date(now).end_of_day + 21.days
+    default_start_date(now).end_of_day + DEFAULT_PERIOD
```

Comment:
I wonder if some of these uses of `DEFAULT_PERIOD` should be using `period_days` instead? As it is, they seem to assume that there isn't an EIA.